src/index.js

```diff
@@ -11,7 +11,4 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
```

Comment:
- [ ] You can go ahead and delete all the boilerplate code that got created by the CRA (texts, images, and styles). Like these here. 
- [ ] Don't forget the files too and images. You won't be needing them for your project

Reply:
Addressed this but you can just keep `index.html` in the `public` folder I think that what you want for now